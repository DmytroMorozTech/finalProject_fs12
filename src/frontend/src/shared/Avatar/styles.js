import { makeStyles } from '@material-ui/core/styles'

export default makeStyles(() => ({

  avatar: {
    color: 'grey',
    fontSize: 55,
    cursor: 'pointer'
  },
  userAvatar: {
    width: '65px',
    borderRadius: '50%'
  }
}))