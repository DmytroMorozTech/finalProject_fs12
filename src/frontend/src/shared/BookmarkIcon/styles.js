import {makeStyles} from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  bookmark: {
    color: theme.palette.grey[700],
    fontSize: theme.typography.icons.medium.fontSize,
    cursor: 'pointer'
  }
}))
