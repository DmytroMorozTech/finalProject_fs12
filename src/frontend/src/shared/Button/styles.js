import {makeStyles} from '@material-ui/core/styles'

export default makeStyles(() => ({

  btn: {
    margin: '5px',
    borderRadius: 30,
    boxShadow: 'none',
    paddingTop: '2px',
    paddingBottom: '2px',
    '&:hover': {
      boxShadow: 'none'
    },

    '&:disabled': {
      cursor: 'not-allowed',
      pointerEvents: 'auto'
    }
  }
}))
