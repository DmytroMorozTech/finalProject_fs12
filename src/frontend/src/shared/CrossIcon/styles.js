import {makeStyles} from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  cross: {
    display: 'flex',
    width: theme.spacing(6),
    color: 'rgba(0,0,0,.5)',
    fontSize: theme.typography.icons.large.fontSize,
    cursor: 'pointer'
  }
}))
