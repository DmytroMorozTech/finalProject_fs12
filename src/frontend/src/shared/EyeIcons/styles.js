import {makeStyles} from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  eye: {
    display: 'flex',
    justifyContent: 'center',
    width: theme.spacing(5)
  },

  eyes: {
    color: theme.palette.grey[500],
    fontSize: theme.typography.icons.medium.fontSize,
    cursor: 'pointer'
  },

  hidden: {
    display: 'none'
  }
}))
