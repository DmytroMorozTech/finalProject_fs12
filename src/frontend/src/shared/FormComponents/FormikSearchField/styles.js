import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({
  searchFieldWrapper: {
    position: 'relative',
    zIndex: 100
  },

  searchDropDownWrapper: {
    width: '100%',
    position: 'absolute',
    top: '38px',
    left: 0
  },

  dropDownItem: {
    width: '100%',
    backgroundColor: 'white',
    height: 40,
    zIndex: 100,
    border: '1px solid gray',
    '&:hover': {
      backgroundColor: 'lightgray',
      cursor: 'pointer'
    }
  },

  hidden: {
    display: 'none'
  }

}))
