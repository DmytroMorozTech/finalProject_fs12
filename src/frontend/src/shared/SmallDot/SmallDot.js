import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord'
import React from 'react'
import styles from './styles'

function SmallDot () {
  const classes = styles()
  return (
    <div className={classes.smallDot}>
      <FiberManualRecordIcon fontSize="inherit"/>
    </div>
  )
}

export default SmallDot
