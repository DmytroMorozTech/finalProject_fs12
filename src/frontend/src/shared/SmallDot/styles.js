import { makeStyles } from '@material-ui/core/styles'

export default makeStyles(() => ({

  smallDot: {
    display: 'flex',
    justifyContent: 'center',
    margin: '5px',
    '& > .MuiSvgIcon-root': {
      fontSize: 3
    }
  }
}))
