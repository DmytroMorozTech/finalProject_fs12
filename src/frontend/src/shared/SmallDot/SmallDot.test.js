import React from 'react'
import SmallDot from './SmallDot'
import { render } from 'enzyme'
import theme from '../../Theme/Theme'
import { ThemeProvider } from '@material-ui/styles'

const setUp = () => render(
  <ThemeProvider theme={theme}>
    <SmallDot />
  </ThemeProvider>)

describe('should render SmallDot component', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
