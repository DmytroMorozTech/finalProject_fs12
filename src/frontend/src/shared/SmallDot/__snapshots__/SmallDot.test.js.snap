// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SmallDot component to match snapshot 1`] = `
<div
  class="makeStyles-smallDot-1"
>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <circle
      cx="12"
      cy="12"
      r="8"
    />
  </svg>
</div>
`;
